src/main/java/dev/j3fftw/litexpansion/machine/AdvancedSolarPanel.java

```diff
@@ -153,7 +193,7 @@ public int getCapacity() {
                 Items.CARBON_PLATE, new ItemStack(Material.LAPIS_BLOCK), Items.CARBON_PLATE,
                 Items.IRIDIUM_PLATE, Items.ADVANCED_MACHINE_BLOCK, Items.IRIDIUM_PLATE,
                 SlimefunItems.ADVANCED_CIRCUIT_BOARD, Items.IRIDIUM_PLATE, SlimefunItems.ADVANCED_CIRCUIT_BOARD
-            }),
+        }),
```

Comment:
dunno wut i did here will it mess stuff up?

Reply:
Please fix